homeassistant/components/zwave_js/light.py

```diff
@@ -85,27 +85,31 @@ def __init__(
         """Initialize the light."""
         super().__init__(config_entry, client, info)
         self._supports_color = False
-        self._supports_white_value = False
+        self._supports_rgbw = False
         self._supports_color_temp = False
         self._hs_color: tuple[float, float] | None = None
-        self._white_value: int | None = None
+        self._rgbw_color: tuple[int, int, int, int] | None = None
+        self._color_mode: str | None = None
         self._color_temp: int | None = None
         self._min_mireds = 153  # 6500K as a safe default
         self._max_mireds = 370  # 2700K as a safe default
-        self._supported_features = SUPPORT_BRIGHTNESS
+        self._supported_color_modes = set()
+        self._supported_features = 0
 
         # get additional (optional) values and set features
         self._target_value = self.get_zwave_value("targetValue")
         self._dimming_duration = self.get_zwave_value("duration")
         if self._dimming_duration is not None:
             self._supported_features |= SUPPORT_TRANSITION
         self._calculate_color_values()
-        if self._supports_color:
-            self._supported_features |= SUPPORT_COLOR
+        if self._supports_rgbw:
+            self._supported_color_modes.add(COLOR_MODE_RGBW)
+        elif self._supports_color:
+            self._supported_color_modes.add(COLOR_MODE_HS)
```

Comment:
zwave is using raw RGB internally, we might want to flag COLOR_MODE_RGB instead.
Also, if the light supports both color and color_temp, it's a 5 channel light and we might want to flag COLOR_MODE_RGBWW as well.